### ISSUE TITLE
KeyError: 'traffic' in get_systems

### ISSUE THREAD

**Issue by huangw5** [djtimca/googlewifi-api#3]
In a recent Google API update, I am seeing the following error:

```
Traceback (most recent call last):
  File "/home/huangwe/Projects/github.com/huangw5/interrupt-wifi-bot/interrupt-wifi-bot.py", line 260, in <module>
    main()
  File "/home/huangwe/Projects/github.com/huangw5/interrupt-wifi-bot/interrupt-wifi-bot.py", line 174, in main
    bot = Bot(args.wifi_token)
  File "/home/huangwe/Projects/github.com/huangw5/interrupt-wifi-bot/interrupt-wifi-bot.py", line 67, in __init__
    for system in asyncio.run(gw.get_systems()).values():
  File "/usr/lib/python3.7/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.7/asyncio/base_events.py", line 584, in run_until_complete
    return future.result()
  File "/home/huangwe/.local/lib/python3.7/site-packages/googlewifi-0.0.17-py3.7.egg/googlewifi/__init__.py", line 190, in get_systems
  File "/home/huangwe/.local/lib/python3.7/site-packages/googlewifi-0.0.17-py3.7.egg/googlewifi/__init__.py", line 290, in structure_systems
KeyError: 'traffic'
```

Looks like the key was removed in the response.

**Comment by Alfiegerner** [comment]
Seeing same setting up homassistant googlewifi integration, can't load devices:

```
This error originated from a custom integration.

Logger: custom_components.googlewifi
Source: custom_components/googlewifi/__init__.py:171
Integration: Google WiFi (documentation, issues)
First occurred: 13:40:28 (10 occurrences)
Last logged: 13:49:44

Unexpected error fetching GoogleWifi data: 'traffic'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 187, in _async_refresh
    self.data = await self._async_update_data()
  File "/config/custom_components/googlewifi/__init__.py", line 171, in _async_update_data
    system_data = await self.api.get_systems()
  File "/usr/local/lib/python3.9/site-packages/googlewifi/__init__.py", line 191, in get_systems
    return await self.structure_systems(response)
  File "/usr/local/lib/python3.9/site-packages/googlewifi/__init__.py", line 291, in structure_systems
    devices[this_station["station"]["id"]]["traffic"] = this_station["traffic"]
KeyError: 'traffic'
```

**Comment by djtimca** [comment]
Fixed in 0.0.18
